source/localizable/models/customizing-serializers.md

```diff
@@ -652,25 +652,52 @@ If you would like to create a custom serializer its recommend that you
 start with the `JSONAPISerializer` or `JSONSerializer` and extend one of
 those to match your needs. However, if your payload is extremely
 different from one of these serializers you can create your own by
-extending the `DS.Serializer` base class. There are 3 methods that
-must be implemented on a serializer.
+extending the `DS.Serializer` base class.
 
-- [normalizeResponse](http://emberjs.com/api/data/classes/DS.Serializer.html#method_normalizeResponse)
-- [serialize](http://emberjs.com/api/data/classes/DS.Serializer.html#method_serialize)
-- [normalize](http://emberjs.com/api/data/classes/DS.Serializer.html#method_normalize)
 
-Its also important to know about the `normalized` JSON form that Ember
-Data expects as an argument to `store.push()`.
 
-`store.push` accepts a JSON API document. However, unlike the
-JSONAPISerializer, `store.push` does not do any transformation of the
-record's type name or attributes. It is important to make sure that
-the type name matches the name of the file where it is defined
-exactly. Also attribute and relationship names in the JSON API
-document should match the name and casing of the attribute and
-relationship properties on the Model.
+A serializer in Ember Data is responsible for normalizing a payload
+from an adapter into the format that Ember Data understand. It is also
+responsible for transforming a snapshot of a record into the payload
+that an adapter will send to the backend.
 
-For Example: given this `post` model.
+A serializer has two main roles in Ember Data. First it is
```

Comment:
You mention two main roles but I'm not clearly seeing a second main role mentioned after this paragraph anywhere.  Am I missing it somewhere?